db/sample_data/example-flu-campaign.json

```diff
@@ -1201,5 +1201,92 @@
         }
       ]
     }
+  ],
+  "registrations": [
+    {
+      "firstName": "Chasidy",
```

Comment:
This reminds me that we really outta tidy up the rest of the snake-case in these JSON files.

Reply:
Tidy up in what way? Isn't it all camel cased? That's idiomatic in JS/JSON, I think we should leave it as is.